tests/container_disk_test.go

```diff
@@ -25,6 +25,10 @@ import (
 	"strings"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
```

Comment:
Could you move these back with the other `kubevirt.io/kubevirt/tests`?

Reply:
Done